dashboard/app/views/layouts/_font_awesome_pro.html.haml

```diff
@@ -0,0 +1,2 @@
+-# Import FONT AWESOME PRO for all dashboard pages
+%script{src: "https://kit.fontawesome.com/ea2d9d4413.js", crossorigin: "anonymous", async: true}
```

Comment:
This takes a dependency on a third-party domain, which we should avoid doing, especially for something so essential.  We should self-host all related content for this change.

Reply:
Good point, thank you! I've found a way to import font awesome pro kit via ruby, just let me find the token for our kit and I'll update the PR